### DISCUSSION TITLE
My CSS not being applied in pages

### DISCUSSION THREAD

**Discussion by Laura-star20** [community/community#23671] [Other Feature Feedback, Questions, & Ideas]
<p>Hello eveyone,</p>
<p>I am new to github. I’ve been following tutoials etc on how to set up pages. Now, i’m trying to move my codepen HTML and CSS over here. HTML is working, but CSS is not being applied.</p>
<p>I’m obviously doing something wrong. But I can’t figure it out.</p>
<aside class="onebox allowlistedgeneric">
  <header class="source">
      <a href="https://laura-star20.github.io/MOD2_SOLUTION/" target="_blank" rel="noopener nofollow ugc">laura-star20.github.io</a>
  </header>
  <article class="onebox-body">
    <img src="" class="thumbnail" width="" height="">

<h3><a href="https://laura-star20.github.io/MOD2_SOLUTION/" target="_blank" rel="noopener nofollow ugc">SOLUTION MODULE 2</a></h3>

  </article>
  <div class="onebox-metadata">
    
    
  </div>
  <div style="clear: both"></div>
</aside>

<p>Thank you in advance for any pointers.</p>
<p>Laura</p>

**Comment by kingthorin** [comment]
<p>Paths/file name are case sensitive.</p>

**Comment by thinkverse** [comment, marked as answer]
<p>Hi there.  :wave:</p>
<p>As <a class="mention" href="/kingthorin">@kingthorin</a> mentioned, files and paths are case-sensitive in GitHub pages.</p>
<p>Meaning, to get your CSS to work you need to change your <code>&lt;link&gt;</code> to.</p>
<pre><code class="lang-auto">&lt;link rel="stylesheet" type="text/css" href="STYLES/STYLE.CSS"&gt;
</code></pre>
<p>Or another way is to lowercase the directory name and file name.</p>
<p>Hope that helps. :+1:</p>
